charts/app/templates/lpaQuestionnaireWebApp.yaml

```diff
@@ -49,6 +49,8 @@ spec:
               path: /health
               port: {{.Values.lpaQuestionnaireWebApp.service.name }}
           env:
+            - name: ALLOW_CREATE
```

Comment:
@davehlewis should this match the one in docker-compose.yml, ALLOW_APPEAL_REPLY_CREATE?

Reply:
yes of course, thanks